src/lib.rs

```diff
@@ -608,6 +616,36 @@ impl<Types: IpfsTypes> Ipfs<Types> {
         rx.await?
     }
 
+    // Initiate a query for random key to discover peers.
+    pub async fn bootstrap(&self) -> Result<(), Error> {
+        let (tx, rx) = oneshot_channel::<Result<FutureSubscription<(), String>, Error>>();
+
+        self.to_task.clone().send(IpfsEvent::Bootstrap(tx)).await?;
+
+        rx.await??.await?.map_err(|e| anyhow!(e))
+    }
+
+    pub async fn add_peer(&self, peer_id: PeerId, addr: Multiaddr) -> Result<(), Error> {
```

Comment:
Could this have a method comment? This is basically to add a known or bootstrap node at runtime?

Reply:
yeah, will do - and yes, this adds a known node to Kademlia at runtime